rate-limit-cloud-controller-api.html.md.erb

```diff
@@ -42,6 +43,22 @@ This might result in inconsistent values for the `X-RateLimit-Remaining` header
 
 When requests exceed the maximum rate limit value, CAPI returns a `429: Too Many Requests` error code.
 
+
+## <a id="v2-api"></a> Rate Limit Responses: V2 API
+
+Operators can optionally limit the number of V2 API requests. This includes all `v2/*` requests except `v2/info`. Each HTTP header includes the following information:
+
+```
+X-Ratelimit-Limit-V2-Api: 60
+X-Ratelimit-Remaining-V2-Api: 56
+X-Ratelimit-Reset-V2-Api: 1643767322
+```
+
+V2 API requests are counted identically to the general rate limit and the headers contain the same information. See table above for more details.
```

Comment:
Maybe something more like:

"_The way V2 API requests are counted is the same as for the general rate limiter, and headers..._"